app/src/components/toggleTip/toggleTip.tsx

```diff
@@ -0,0 +1,161 @@
+import { useState, useRef, useEffect } from "react";
+import { Text, Box, Icon } from "@nypl/design-system-react-components";
+import { Button } from "@chakra-ui/react";
+
+export const ToggleTip = ({
+  text,
+  toggleTipContent,
+}: {
+  text: string;
+  toggleTipContent: string;
+}) => {
+  const [isVisible, setIsVisible] = useState(false);
+  const liveRegionRef = useRef<HTMLSpanElement | null>(null);
+  const buttonRef = useRef<HTMLButtonElement | null>(null);
+
+  const addLiveRegion = () => {
+    if (liveRegionRef.current) {
+      setTimeout(() => {
+        liveRegionRef.current!.textContent = toggleTipContent;
+      }, 200);
+    }
+  };
+
+  const removeLiveRegion = () => {
+    if (liveRegionRef.current) {
+      setTimeout(() => {
+        liveRegionRef.current!.textContent = "";
+      }, 200);
+    }
+  };
+
+  const handleClick = (event: MouseEvent | KeyboardEvent) => {
+    if (
+      buttonRef.current &&
+      !buttonRef.current.contains(event.target as Node) &&
+      isVisible
+    ) {
+      setIsVisible(false);
+      removeLiveRegion();
+    }
+  };
+
+  const handleBlur = (event: React.FocusEvent) => {
+    if (!buttonRef.current?.contains(event.relatedTarget as Node)) {
+      setIsVisible(false);
+      removeLiveRegion();
+    }
+  };
+
+  useEffect(() => {
+    const handleEscPress = (event: KeyboardEvent) => {
+      if (event.key === "Escape" && isVisible) {
+        setIsVisible(false);
+        removeLiveRegion();
+      }
+    };
+
+    document.addEventListener("mousedown", handleClick);
+    document.addEventListener("keydown", handleEscPress);
+
+    return () => {
+      document.removeEventListener("mousedown", handleClick);
+      document.removeEventListener("keydown", handleEscPress);
+    };
+  }, [isVisible]);
+
+  return (
+    <Box as="span" display="inline-flex" alignItems="center">
+      {text}
+      <Box position="relative" display="inline-block">
+        <Button
+          ref={buttonRef}
+          aria-label="More info"
+          lineHeight="1"
+          background="unset"
+          padding="0 !important"
+          marginBottom="xxs"
+          marginLeft="xxs"
+          height="unset"
+          minWidth="unset"
+          _hover={{
+            background: "unset",
+            outline: "unset",
+          }}
+          _focus={{ outline: "2px solid var(--nypl-colors-ui-link-primary)" }}
+          onClick={() => {
+            setIsVisible((prev) => !prev);
+            if (!isVisible) {
+              addLiveRegion();
+            } else {
+              removeLiveRegion();
+            }
+          }}
+          onFocus={() => {
+            if (!isVisible) {
```

Comment:
I don't think you need this conditional here. It will always be false before the button is focused and you always want to set it visible on focus, right?

Reply:
However, from Clare's feedback, it shouldn't be visible on focus but rather when the user presses enter or space (which does work to close it).